app-hub-gui/src/data.ts

```diff
@@ -8,25 +13,78 @@ interface CodeCoverageData {
 export class CCData {
   constructor() {}
 
-  GetCCData(): CodeCoverageData[] {
-    return [
-      {
-        name: "app1",
-        coverage: 0,
-        date: "n/a"
-      },
-      {
-        name: "App 2",
-        coverage: 80,
-        date: "2019/03/04 12:15"
-      },
+  async GetCCData(): Promise<CodeCoverageData[]> {
+    //copied from AppHubApi
+    const typeDefs = gql`
```

Comment:
We should add a todo for this to point to the api.... if this is actually doing anything..?

Reply:
yeah for sure.